src/ex4/server/db/models/items.cjs

```diff
@@ -0,0 +1,33 @@
+'use strict';
+const {
+  Model
+} = require('sequelize');
+
+var DataTypes = require('sequelize/lib/data-types');
+
+module.exports = (sequelize) => {
+  class items extends Model {
```

Comment:
according to convention we prefer a model to be a single instance, so "item" and not "items".
the name of the table should be "items" though.